assets/js/blocks/mini-cart/block.tsx

```diff
@@ -44,14 +45,16 @@ interface Props {
 	hasHiddenPrice: boolean;
 }
 
-const MiniCartBlock = ( {
-	isInitiallyOpen = false,
-	colorClassNames,
-	style,
-	contents = '',
-	addToCartBehaviour = 'none',
-	hasHiddenPrice = false,
-}: Props ): JSX.Element => {
+const MiniCartBlock = ( attributes: Props ): JSX.Element => {
+	const {
+		isInitiallyOpen = false,
+		colorClassNames,
+		style,
+		contents = '',
+		addToCartBehaviour = 'none',
+		hasHiddenPrice = false,
+	} = attributes;
```

Comment:
Just curious, why did you change the destructuring to be within the function body? To be clear, I also like it more, I wonder if there is a specific reason.

Reply:
Good question!

It was adjusted so that we could access the whole `attributes` object directly for use in the `useTyporgraphyProps` hook[ below](https://github.com/woocommerce/woocommerce-blocks/pull/6760/files#diff-733abdc48b34d465d994c1bf4b179fdb74a9e061ae5bc020bd0f5db954a7b267R207).